sentinel-core/src/main/java/com/alibaba/csp/sentinel/traffic/TrafficRuleSubscriber.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 1999-2018 Alibaba Group Holding Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.alibaba.csp.sentinel.traffic;
+
+/**
+ * The subscriber of traffic rule.
+ * @author panxiaojun233
+ */
+public interface TrafficRuleSubscriber {
```

Comment:
Here we actually observes the event of "remote provider app change", not the rule. So maybe `RemoteAppObserver` or other name is better?

The same for `subscribe`/`unsubscribe`.

Reply:
done